src/communication/network_interface_cl.h

```diff
@@ -125,6 +125,7 @@ extern int file_apply_tde_to_class_files (const OID * class_oid);
 #ifdef UNSTABLE_TDE_FOR_REPLICATION_LOG
 extern int tde_get_data_keys ();
 #endif /* UNSTABLE_TDE_FOR_REPLICATION_LOG */
+extern int get_dblink_chpher_master_key ();
```

Comment:
how about adding a prefix to this function's name? other functions have prefixes to easily know which module calls it
e.g) `dblink_get_cipher_master_key();`

Reply:
I will accept your comments and fix it.